app/src/main/java/com/locked/shingranicommunity/event/EventListViewModel.kt

```diff
@@ -72,8 +81,107 @@ class EventListViewModel @Inject constructor(
 //        return itemEventHandler.rejected(rsvp!!,item._id)
 //    }
 
+    // use this onBind() of List Adapter
+    fun getItemViewModel(itemIndex: Int): ItemViewModel? {
```

Comment:
use this function in `onBindViewHolder()` of `EventListAdapter`. It will give you a `ItemViewModel`. You will set it to the `EventViewHolder`. Inside the `EventViewHolder` you will observe each of viewmodel's property and update your view accordingly.

Reply:
@ns-jeeb